snap/snapcraft.yaml

```
@@ -129,6 +129,7 @@ parts:
       - flex
       - bison
     source: https://codeberg.org/dnkl/yambar.git
+    source-tag: 1.9.0
```

Tiny optimization:

```suggestion
    source-tag: 1.9.0
    source-depth: 1
```

(should be the default, if you ask me).